From e3e60686e15e8f1e19e28997cdcd837bf46b3a94 Mon Sep 17 00:00:00 2001
From: Sandeep Singh <Sandeep@freescale.com>
Date: Mon, 7 May 2012 16:11:56 +0000
Subject: [PATCH 114/121] tdm_test: Changing poll operation from TDM port to
 TDM channel

Signed-off-by: Sandeep Singh <Sandeep@freescale.com>
[Kevin: Original patch taken from FSL
QorIQ-SDK-V1.2-SOURCE-20120614-yocto.iso image]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/tdm/test/tdm_test.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/tdm/test/tdm_test.c b/drivers/tdm/test/tdm_test.c
index 243936e..40ea699 100644
--- a/drivers/tdm/test/tdm_test.c
+++ b/drivers/tdm/test/tdm_test.c
@@ -109,7 +109,7 @@ static int tdm_thread(void *ptr)
 	while ((poll < poll_count) && !kthread_should_stop()) {
 
 		poll++;
-		while (tdm_port_poll(h_port, 10) != TDM_E_OK)
+		while (tdm_ch_poll(h_channel1, 10) != TDM_E_OK)
 				continue;
 
 		ret = tdm_channel_read(h_port, h_channel1, p_data1, &size);
@@ -214,7 +214,7 @@ static int test_detach_adapter(struct tdm_adapter *adap)
 }
 
 static const struct tdm_device_id test_id[] = {
-	{ "fsl", 0 },
+	{ "fsl_tdm", 0 },
 	{ }
 };
 
-- 
1.7.9.7

